Makefile

```diff
@@ -295,3 +298,38 @@ release-manifests: manifests kustomize
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	mkdir -p releases
 	$(KUSTOMIZE) build config/default > releases/aiven-operator-${IMG_TAG}.yaml
+
+.PHONY: e2e-setup-kind
+OPERATOR_TAG ?= $(shell git rev-parse HEAD)
+CERT_MANAGER_TAG ?= v1.9.0
+CERT_MANAGER ?= https://github.com/cert-manager/cert-manager/releases/download/$(CERT_MANAGER_TAG)/cert-manager.crds.yaml
+PREREQUISITES = jq base64 kcat helm kind docker avn
+e2e-setup-kind:
```

Comment:
I think the `e2e-setup-kind` is a bit misleading, we do more that just setting up kind. Do you think it is possible to combine this actions with the  `test-e2e-preinstalled` into one actions?

Reply:
e2e setups in kind starting with helm, so it is from one end to another.  
I wouldn't combine that with tests, because you might want to run tests against GKE cluster or something.